client/src/components/AuditAdmin/AuditAdminView.tsx

```diff
@@ -141,7 +141,7 @@ const AuditAdminView: React.FC = () => {
               <Progress
                 jurisdictions={jurisdictions}
                 auditSettings={auditSettings}
-                round={rounds[rounds.length - 1]}
+                round={rounds.length > 0 ? rounds[rounds.length - 1] : null}
```

Comment:
This was an unrelated issue I found where `round` was getting set to `undefined` instead of `null` before the audit started. The code that consumed it had compensated by using `!!round` as opposed to `round !== null`, but I wanted to tighten it up, since they types say that it's `null`.

Reply:
Nice catch!